### ISSUE TITLE
Game not unpausing when entering microbe editor

### ISSUE THREAD

**Issue by stoptakingusernames** [Revolutionary-Games/Thrive#3440]
**Describe the bug**

The game will no longer unpause if you enter the microbe editor while paused, which causes you to be unable to add any organelles to your cell and replaces the membrane with a white square in the membrane editor. seems to be a regression since last release

**To Reproduce**
Steps to reproduce the behaviour:
1. Start a new game and gather the required amount of compounds needed to reproduce
2. Pause the game with either the hotkey or the pause button
3. Enter the microbe editor

**Expected behaviour**
The game should unpause right after clicking the reproduction button, as it did in 0.5.8.1

**Additional context**
This bug does not occur in version 0.5.8.1

**Comment by Kasterisk** [comment]
Can confirm, happens in multicellular too. Seems like a regression from https://github.com/Revolutionary-Games/Thrive/commit/2fc3f4d82c4de0e6d906a91f62f8d9f2b832c62c.

**Comment by hhyyrylainen** [comment]
I may have fixed this in my prototype branch as I refactored stage functionality and had to reapply the done changes from that other PR. So before anyone jumps on this, please check my branch first. I'll also try to remember to check this in my branch before I merge it.

**Comment by Oliveriver** [comment]
I've noticed some other pause weirdness related to becoming mutlicellular. The game doesn't pause when the multicellular popup appears, but the pause button does switch to its pressed state and stays there the next time you return to the environment.

I checked and it is a regression caused by https://github.com/Revolutionary-Games/Thrive/commit/0bf60d145ae0ff95274acdbf4503d5f7c8a2a120, so the other issue here is likely to have been caused by this too.
